sympy/polys/multivariate_resultants.py

```diff
@@ -0,0 +1,389 @@
+    [-1,  0,  0,  1,  0]])
+    >>> matrix.det()
+    0
+
+    Reference
+    ==========
+    1. [Kapur1994]_
+    2. [Palancz08]_
+
+    See Also
+    ========
+    Notebook in examples: sympy/example/notebooks.
+    """
+
+    def __init__(self, polynomials, variables):
+        """
+        A class that takes two lists, a list of polynomials and list of
+        variables. Returns the Dixon matrix of the multivariate system.
+
+        Parameters
+        ----------
+        polynomials : list of polynomials
+            A  list of m n-degree polynomials
+        variables: list
+            A list of all n variables
+        """
+        self.polynomials = polynomials
+        self.variables = variables
+
+        self.n = len(self.variables)
+        self.m = len(self.polynomials)
+
+        self.dummy_variables = self.get_dummy_variables()
+        self.max_degrees = self.get_max_degrees()
+
+    def get_dummy_variables(self):
+        """
+        Returns
+        -------
+
+        dummy_variables: list
+            A list of n alpha variables. These are the replacing variables
+        """
+        a = IndexedBase("alpha")
+        dummy_variables = [a[i] for i in range(self.n)]
+
+        return dummy_variables
+
+    def get_max_degrees(self):
+        r"""
+        Returns
+        -------
+
+        degrees: list
+            A list of the d_max of each variable. The max degree is the
+            max(degree(p_1, x_i), ..., degree(p_m, x_i))
+        """
+        return [poly_from_expr(f, self.variables)[0].degree() for f in self.polynomials]
+
+    def get_dixon_polynomial(self):
+        r"""
+        Returns
+        -------
+
+        dixon_polynomial: polynomial
+            Dixon's polynomial is calculated as:
+
+            delta = Delta(A) / ((x_1 - a_1) ... (x_n - a_n)) where,
+
+            A =  |p_1(x_1,... x_n), ..., p_n(x_1,... x_n)|
+                 |p_1(a_1,... x_n), ..., p_n(a_1,... x_n)|
+                 |...             , ...,              ...|
+                 |p_1(a_1,... a_n), ..., p_n(a_1,... a_n)|
+        """
+        if self.m != (self.n + 1):
+            raise ValueError('Method invalid for given combination.')
+
+        # first row
+        rows = [self.polynomials]
+
+        temp = list(self.variables)
+        iterator = iter(self.dummy_variables)
+
+        for idx in range(self.n):
+            temp[idx] = next(iterator)
+            substitution = {self.variables[i]: temp[i] for i in range(len(temp))}
+            rows.append([f.subs(substitution) for f in self.polynomials])
+
+        A = Matrix(rows)
+        product_of_differences = Mul(*[a - b for a, b in zip(self.variables,
+                                                         self.dummy_variables)])
+        dixon_polynomial = (A.det() / product_of_differences).factor()
+
+        return poly_from_expr(dixon_polynomial, self.dummy_variables)[0]
+
+
+    def get_coefficients_of_alpha(self, polynomial):
+        r"""
```

Comment:
How does this compare with `polynomial.coeffs()`?

Reply:
It does not, I should be using `polynomial.coeffs()`. Apologises I missed that.